### ISSUE TITLE
Add an option to disable datapack reloading

### ISSUE THREAD

**Issue by NotroDev** [JorelAli/CommandAPI#552]
### Description

It would be really helpful if there was a way to disable datapack reloading.
Currently, CommandAPI reloads datapacks every time the server starts, and for my server, this process takes around 10 seconds, even though I don't use datapacks or anything related to them.

**Comment by JorelAli** [comment]
Implemented for 9.4.1, due to release later today.

**Comment by JorelAli** [comment]
Fixed in 9.4.1.

**Comment by NotroDev** [comment]
Hey @JorelAli, I don't know if it's possible to contribute to the docs, so I'm writing here:
![docs](https://github.com/JorelAli/CommandAPI/assets/77281214/b386a11f-a195-4e0d-bb86-055689311433)

I think it should be `true` 😄

**Comment by JorelAli** [comment]
Whoops. Contributing to the documentation is possible, but updating the published documentation isn't something we can do very quickly or easily. I've made a commit that fixes this, but this change won't be propagated into the main documentation until the next release.

_(We do have some [live documentation](https://commandapi-live-docs.jorel.dev/config.html#skip-initial-datapack-reload) which includes the current state of development and we may consider using as our main documentation on our website in the future. I've fixed this there for now)_
